Lib/fontTools/merge.py

```diff
@@ -780,6 +849,7 @@ def merge(self, fontfiles):
 		# TODO Is it necessary to reload font?  I think it is.  At least
 		# it's safer, in case tables were loaded to provide glyph names.
 		fonts = [ttLib.TTFont(fontfile) for fontfile in fontfiles]
+		self.fonts = copy.deepcopy(fonts)
```

Comment:
> deepcopy'ing fonts is not a supported operation. Why is this needed?

@behdad An original copy of font is needed since the glyphs in the fonts are read from isGlyphSame(). The variable 'fonts' doesn't work since its glyphs name is changed by line 854: font.setGlyphOrder(glyphOrder).

Otherwise do you think reopening the fonts again a good choice?

sth like:
fonts = [ttLib.TTFont(fontfile) for fontfile in fontfiles]
self.fonts = [ttLib.TTFont(fontfile) for fontfile in fontfiles]